Whisper.net/WhisperFactory.cs

```diff
@@ -37,6 +38,7 @@ private WhisperFactory(IWhisperProcessorModelLoader loader, bool delayInit)
         if (!delayInit)
         {
             var nativeContext = loader.LoadNativeContext(libraryLoaded.Value.NativeWhisper!);
+            isEagerlyInitialized = true;
             contextLazy = new Lazy<IntPtr>(() => nativeContext);
```

Comment:
Wondering whether can just initialize with the context as value.
```suggestion
            var nativeContext = loader.LoadNativeContext(libraryLoaded.Value.NativeWhisper!);
            contextLazy = new Lazy<IntPtr>(nativeContext);
```
Seems that overload isn't an option in .net standard 2.0 though.

Reply:
Good point, we can use that overload for net8 and greater to not allocate that delegate anymore + creating the closure.

Adapted the code to use it and still use the overload with the delegate for netstandard. However, we'll still require that isEagerlyInitialized for those cases.